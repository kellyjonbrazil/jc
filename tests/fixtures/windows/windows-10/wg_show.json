[
  {
    "device": "wg0",
    "privateKey": "Xa4Bq8AUFrjKp+Z283VDzEUVeOOn2wX/uuLTMUkkJens=",
    "publicKey": "mL2Co/VecYWtXSY6YUZAI6cAsFZDCEV+9iZKibNcWTu=",
    "listenPort": 45321,
    "fwmark": null,
    "peers": [
      {
        "publicKey": "bN9QoMEqlKUpncq0Q8ye9yURczKwmbXzwVsG7A+RV2B=",
        "presharedKey": null,
        "endpoint": "123.45.67.89:51820",
        "latestHandshake": 0,
        "transferRx": 0,
        "transferSx": 1532,
        "persistentKeepalive": -1,
        "allowedIps": [
          "0.0.0.0/0"
        ]
      }
    ]
  },
  {
    "device": "wg0",
    "privateKey": "yJ3PoNAqlNUpmcq0P7xv8zURdyKwmbZwwUsG9B+SV3C=",
    "publicKey": "kP2Lo/WfgVWtYSY7ZUZAJ7cBsEZECFV+8jYKidOcVTv=",
    "listenPort": 53211,
    "fwmark": null,
    "peers": [
      {
        "publicKey": "dQ5AoNAqpOUlncq0P9ze8xURayLwmbYwwVsH8C+TV4D=",
        "presharedKey": null,
        "endpoint": "203.0.113.4:51820",
        "latestHandshake": 0,
        "transferRx": 0,
        "transferSx": 2001,
        "persistentKeepalive": -1,
        "allowedIps": [
          "192.168.1.0/24"
        ]
      }
    ]
  },
  {
    "device": "wg0",
    "privateKey": "pR6DoOBqmNUlncq0P1wq9zUSeyJwmbVxxWsF7D+RU5E=",
    "publicKey": "qN2Mo/XghWWtXSY8YVZBK8cDsFZDCFQ+7kZKjbNcWTx=",
    "listenPort": 67514,
    "fwmark": null,
    "peers": [
      {
        "publicKey": "tA8AoPCqoPUlncq0P8xe7xUSayKwmbWxxVsG8E+UV6F=",
        "presharedKey": null,
        "endpoint": "198.51.100.15:51820",
        "latestHandshake": 0,
        "transferRx": 0,
        "transferSx": 1403,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.0.0.0/8"
        ]
      }
    ]
  },
  {
    "device": "wg0",
    "privateKey": "zU9CoQCqmQUlncq0P4wo6yUTbyJwmbYyyWsF9F+SV7G=",
    "publicKey": "vM2Ko/YigXWtYSY9YUZCJ9dCsGZECFV+5lZLjcOcVVy=",
    "listenPort": 79812,
    "fwmark": null,
    "peers": [
      {
        "publicKey": "xC7DoRDqnRUlncq0P2wq9yUScyKwmbXzyWsH8F+TU8H=",
        "presharedKey": null,
        "endpoint": "172.16.0.1:51820",
        "latestHandshake": 0,
        "transferRx": 0,
        "transferSx": 1872,
        "persistentKeepalive": -1,
        "allowedIps": [
          "172.16.0.0/16"
        ]
      }
    ]
  },
  {
    "device": "wg0",
    "privateKey": "rE4AoSCqnSUlncq0P7wv9zURdyLwmbZxzWsG9G+UV9I=",
    "publicKey": "fN2Oo/ZjhYUtXSZ0YUZDK8eDsFZDCFQ+4jYKjcNcVVw=",
    "listenPort": 91114,
    "fwmark": null,
    "peers": []
  }
]
