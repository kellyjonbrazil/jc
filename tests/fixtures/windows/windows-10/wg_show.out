wg0     Xa4Bq8AUFrjKp+Z283VDzEUVeOOn2wX/uuLTMUkkJens=    mL2Co/VecYWtXSY6YUZAI6cAsFZDCEV+9iZKibNcWTu=    45321   off
wg0     bN9QoMEqlKUpncq0Q8ye9yURczKwmbXzwVsG7A+RV2B=    (none)  123.45.67.89:51820      0.0.0.0/0       0       0       1532    off
wg0     yJ3PoNAqlNUpmcq0P7xv8zURdyKwmbZwwUsG9B+SV3C=    kP2Lo/WfgVWtYSY7ZUZAJ7cBsEZECFV+8jYKidOcVTv=    53211   off
wg0     dQ5AoNAqpOUlncq0P9ze8xURayLwmbYwwVsH8C+TV4D=    (none)  203.0.113.4:51820       192.168.1.0/24  0       0       2001    off
wg0     pR6DoOBqmNUlncq0P1wq9zUSeyJwmbVxxWsF7D+RU5E=    qN2Mo/XghWWtXSY8YVZBK8cDsFZDCFQ+7kZKjbNcWTx=    67514   off
wg0     tA8AoPCqoPUlncq0P8xe7xUSayKwmbWxxVsG8E+UV6F=    (none)  198.51.100.15:51820     10.0.0.0/8      0       0       1403    off
wg0     zU9CoQCqmQUlncq0P4wo6yUTbyJwmbYyyWsF9F+SV7G=    vM2Ko/YigXWtYSY9YUZCJ9dCsGZECFV+5lZLjcOcVVy=    79812   off
wg0     xC7DoRDqnRUlncq0P2wq9yUScyKwmbXzyWsH8F+TU8H=    (none)  172.16.0.1:51820        172.16.0.0/16   0       0       1872    off
wg0     rE4AoSCqnSUlncq0P7wv9zURdyLwmbZxzWsG9G+UV9I=    fN2Oo/ZjhYUtXSZ0YUZDK8eDsFZDCFQ+4jYKjcNcVVw=    91114   off
